normandy/recipes/api/filters.py

```diff
@@ -36,7 +36,7 @@ def filter(self, qs, value):
                 raise TypeError("BaselineCapabilitiesFilter can only be used to filter recipes")
             match_ids = []
             for recipe in recipes:
-                if recipe.uses_only_baseline_capabilities():
+                if recipe.latest_revision.uses_only_baseline_capabilities():
```

Comment:
I'm not sure about this one either. I think it probably doesn't matter much what it does since we don't use it when e.g. publishing to Remote Settings, but maybe you have a better idea.

Reply:
I think that this should be `approved_revision`. This is primarily used so that the `/api/v1/recipes/signed/` endpoint stays compatible with old clients. Since that endpoint primarily deals with approved revisions, I think this filter should as well.